include/Giraffe.hpp

```diff
@@ -3,15 +3,19 @@
 
 #include <utility>
 
+#include "GiraffeText.hpp"
 #include "Util/GameObject.hpp"
 #include "Util/Text.hpp"
 
 class Giraffe : public Util::GameObject {
 
 public:
-    void Update(const Util::Transform &transform = Util::Transform()) override;
+    void Update();
 
-    void Start() override;
+    void Start();
```

Comment:
if the implementation is empty maybe this could be removed

Reply:
Shoot, forgot to remove that.